build.sbt

```
@@ -4,27 +4,33 @@ lazy val scala_212 = "2.12.13"
 lazy val scala_213 = "2.13.4"
 
 lazy val V = new {
-  val avro4s        = "4.0.4"
-  val cats          = "2.4.2"
-  val `cats-effect` = "2.4.0"
-  val ciris         = "1.2.1"
-  val confluent     = "6.1.0"
-  val doobie        = "0.12.1"
-  val kafka         = "2.7.0"
-  val logback       = "1.2.3"
-  val `log-effect`  = "0.14.1"
-  val ociSdk        = "1.33.2"
-  val postgres      = "42.2.19"
-  val refined       = "0.9.21"
-  val scalacheck    = "1.15.3"
-  val scalatest     = "3.2.6"
-  val silencer      = "1.7.3"
-  val sttp          = "3.1.9"
-  val zio           = "1.0.5"
-  val `zio-interop` = "2.3.1.0"
-  val `zio-kafka`   = "0.14.0"
-  val `zio-oci-os`  = "0.1.3"
-  val `zio-s3`      = "0.3.0"
+  val avro4s       = "4.0.4"
+  val cats         = "2.4.2"
+  val ciris        = "1.2.1"
+  val confluent    = "6.1.0"
```

```suggestion
  val confluent    = "6.1.1"
```
this should be ok, also it will pull in `com.github.everit-org.json-schema:org.everit.json.schema:1.12.2`, which is a patch version later than the one it's trying to pull in now.. the one we have cached (`1.12.1`) has a SHA-1 that's not matching the one on jitpack...